village/src/components/Nav.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { BrowserRouter as Router, Route, Link } from "react-router-dom";
+import SmurfForm from './components/SmurfForm';
+import Smurfs from './components/Smurfs';
+
+const Nav = () => {
```

Comment:
clean Nav component. What was the reasoning to use a functional component over a class component for this item?

Reply:
with class i have to use render then put constructor with props and all the other long stuf but with function is easier and clean straight forward then just return the output